org.nickvision.money.gnome/src/helpers/builder.cpp

```diff
@@ -13,6 +13,7 @@ namespace Nickvision::Money::GNOME
         std::filesystem::path path{ Aura::getActive().getExecutableDirectory() / "ui" / (blueprint + ".ui") };
         if(!std::filesystem::exists(path))
         {
+            fprintf(stderr, "UI file not found at path: %s", path.c_str());
```

Comment:
you can make this C++ instead of C:

```cpp
std::err << "UI file not found at path: " << path << std::endl;
```

Just make sure to add `#include <iostream>` after `#include <filesystem>`